src/lib/services/account-service.ts

```diff
@@ -52,6 +53,10 @@ export class AccountService {
         return this.store.getAccountByPersonalAccessToken(secret);
     }
 
+    async getAdminCount(): Promise<IAdminCount> {
+        return this.store.getAdminCount();
```

Comment:
Here it might be a good place to translate from DB types (snake_case) into API types (camelCase)

Reply:
Addressed in https://github.com/Unleash/unleash/pull/3829/commits/ecc6f81040f6fd619842bae1552515ba26143568 - I preferred to translate it in the store layer so we use the same interface through all the layers.